scripts/generate_openapi_spec.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with REANA; if not, see <http://www.gnu.org/licenses>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""OpenAPI specification file generator script."""
+
+import json
+import os
+import re
+import sys
+
+import click
+from apispec import APISpec
+from flask import current_app
+from flask.cli import with_appcontext
+
+
+# Import your marshmallow schemas here
+# from example_package.schemas import Example_schema,
+
+
+def get_package_version():
+    """Get version string from version.py of package."""
+
+    with open(os.path.join('reana_server', 'version.py'), 'rt') as f:
+        version = re.search(
+            '__version__\s*=\s*"(?P<version>.*)"\n',
+            f.read()
+        ).group('version')
+    return version
+
+
+@click.command()
+@click.option('-n', '--name', default="Reana package",
+              help='Name of package/component', envvar='REANA_PACKAGE_NAME')
+@click.option('-d', '--description', default="API description",
```

Comment:
Better naming suggestions for these environment variables? e.g. PACKAGE_OAPI_VERSION is not that explanatory and without checking it out or without documentation (e.g. line comments) it is immediately clear what VERSION means in this context.